p4-16/spec/P4-16-spec.mdk

```diff
@@ -2208,9 +2208,12 @@ structField
 ~ End P4Grammar
 
 where each `typeRef` is restricted to a bit-string type (fixed or
-variable) or an integer type. This declaration introduces a new
-identifier in the current scope; the type can be referred to using
-this identifier. A header is similar to a `struct` in C,
+variable), an integer type, a boolean type, or a struct that itself contains other 
```

Comment:
make it clear that 'integer type' is actually a fixed-width integer

Reply:
Will do.